javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -53,6 +54,7 @@
 
     public static final int STRICTFP = Modifier.STRICT;
 
+    @Deprecated
```

Comment:
I was unsure myself, just wanted to bring it up.
I'm fine with keeping that (simple) duplication.